x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/downsample/DownsampleActionSingleNodeTests.java

```diff
@@ -477,7 +477,6 @@ public void onFailure(Exception e) {
         assertBusy(() -> assertTrue("In progress rollup did not complete", rollupListener.success), 60, TimeUnit.SECONDS);
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/88800")
```

Comment:
Is this an unrelated unmuting? Looking at the source issue it seems to have already been fixed elsewhere.

Reply:
Yes, we have fixed this issue some time ago. I don't know why this was not muted. 
I unmuted it in this PR rather than submitting a separate one for this.